examples/generative/corrdiff/training/training_loop.py

```diff
@@ -264,12 +283,17 @@ def training_loop(
             g["lr"] = optimizer_kwargs["lr"] * min(
                 cur_nimg / max(lr_rampup_kimg * 1000, 1e-8), 1
             )  # TODO better handling (potential bug)
+            if patch_shape_x != img_shape_x:
```

Comment:
Decaying learning rate should be independent of whether using patching or not.

Reply:
changed